doc/source/operations/rocky-linux-9.rst

```diff
@@ -456,6 +456,13 @@ address them.
 Once happy that the system has been restored to full health, move onto the next
 host or batch or hosts.
 
+9. Enable the hypervisor in Nova again:
+
+   .. code-block:: console
+
+      kayobe playbook run $KAYOBE_CONFIG_PATH/ansible/nova-compute-enable.yml --limit <hostname>
```

Comment:
I wonder why you are the first to see this. Perhaps the service deploy sometimes brings it up enabled, depending on timing?

Reply:
I've only migrated one hypervisor (the rest are waiting for the az team to do), so I'm not sure about the timings. I can edit this to suggest checking if it is disabled first? Although I think running the enable on a hypervisor that's already enabled won't do anything.

Reply:
Right - it should be idempotent